compositeviews/compositeview01/composite_view.json

```diff
@@ -0,0 +1,25 @@
+{
+  "@id": "connectome-composite-view-01",
+  "@type": "CompositeView",
+  "sources": [
+    {
+      "@id": "test-source-01",
+      "@type": "ProjectEventStream"
+    }
+  ],
+  "projections": [
+    "researchproject.json",
```

Comment:
Maybe the things which are not Nexus syntax should be somehow declared. As here: that we include researchproject.json as an external file should be made more clear. Example: "#include<researchproject.json>" or "!include(researchproject.json" or something similar.

Reply:
Ok, this would be like a template syntax then. Do you think we should also see whether an actual template library should be used instead of inventing our own thing?

Reply:
see #29

Reply:
> Ok, this would be like a template syntax then. Do you think we should also see whether an actual template library should be used instead of inventing our own thing?

Of course that would be much cleaner. I can't give you any recommendation on which python templating language to use best for this task though (escaping / encoding is mostly optimized for html e.g. in jinja2).

Reply:
That would be great, just put it here: #29. But I think at the current state we should not over-engineer this, we have enough problems already ;-)